src/compiler/checker.ts

```diff
@@ -38,6 +38,7 @@ namespace ts {
         let Signature = objectAllocator.getSignatureConstructor();
 
         let typeCount = 0;
+        let symbolCount = 0;

```

Comment:
Yes, we were only showing the number of symbols created by the binder and not including the transient symbols created by the checker.